lib/relevantToExcursion.m

```diff
@@ -0,0 +1,23 @@
+%==========================================================================
+%This function takes in a list of objects and returns only those connected
+%to the specified excursionSet. It takes in 3 inputs.
+%
+%inputCell - a cell of input objects.
+%exNo - the number corresponding to the excursion set of interest.
+%labels - the corresponding label hashmap.
+%
+%Authors: Thomas Maullin, Camille Maumet.
+%==========================================================================
+function resultant = relevantToExcursion(inputCell, exNo, labels)
+    
+    counter = 1;
+    resultant = {};
+    %Work out which object belongs to which excursion set.
+    for(i = 1:length(inputCell))
```

Comment:
Could we avoid this loop by using a map object (see also comment https://github.com/incf-nidash/nidmresults-spmhtml/pull/24/files/c380851c0be7c72761aad57ca1024dfa0bcb71d5#r99836920).